deepchem/feat/molecule_featurizers/mol_graph_conv_featurizer.py

```diff
@@ -125,21 +140,32 @@ class MolGraphConvFeaturizer(MolecularFeaturizer):
   This class requires RDKit to be installed.
   """
 
-  def __init__(self, add_self_edges: bool = False):
+  def __init__(self,
+               use_edges: bool = False,
+               use_chirality: bool = False,
+               use_partial_charge: bool = False):
     """
     Parameters
     ----------
-    add_self_edges: bool, default False
-      Whether to add self-connected edges or not. If you want to use DGL,
-      you sometimes need to add explicit self-connected edges.
+    use_edges: bool, default False
+      Whether to use edge features or not.
```

Comment:
I removed `add_self_edges` and added `use_edges ` flag.
This is for simple GCN or GAT models which don't need edge features.